docs/getting-started/accessing-your-fleet.md

```diff
@@ -0,0 +1,45 @@
+# Accessing Your Fleet
+
+In order to complete Fleet operations, you need to access the **aux node**. From here you can run the `fleet` commands.
+
+## Log in to the aux node
+
+Test you can log into the **aux node**.
+
+```bash
+ssh -t deploy@aux.{fleet-id}.f.nchr.io
+```
+
+An easier way to access fleet is to set up an alias.
+Future examples will assume you have aliased the `fleet` command
+
+```bash
+alias fleet='ssh -t -o LogLevel=QUIET deploy@aux.{fleet-id}.f.nchr.io --'
+```
+
+Once you're done with the test and alias setup above, you should be able to type the following in your terminal and get the list of available commands
+
+```bash
+fleet -h
+```
+
+Should you need to add access to other people in your team, you can add their ssh keys with the following guide. If not, we'll move on to [configuring revision control](getting-started/configuring-revision-control/) so your fleet can access it.
```

Comment:
Yeah, the thing is the following section after that kinda doesn't make sense otherwise. It's a section about adding SSH key, of which they already have theirs in.

It's kinda important thing if they want to have other people accessing their account so that's why I added the "bridging" paragraph.

If you don't like the link, then I can update into something like: If not, move on to the next section